lib/modules.js

```diff
@@ -22,33 +22,27 @@ const path = require('path')
 const _ = require('lodash')
 const semver = require('semver')
 
-function modulesWithKeyword (app, keyword) {
-  var res = []
-
-  var prioritize = (a, b) => {
-    return a.module.startsWith('@signalk') ? -1 : 1
-  }
-
-  if (app.config.configPath != app.config.appPath) {
-    var modulesPath = path.join(app.config.configPath, 'node_modules')
+const priorityPrefix = (a, b) =>
+  a.module
+    .replace('@signalk', ' ')
+    .localeCompare(b.module.replace('@signalk', ' '))
 
+function modulesWithKeyword (app, keyword) {
+  const modules = new Set()
+  function searchAddModules (searchPath) {
+    debug('searchPath: ' + searchPath)
+    const modulesPath = path.join(searchPath, 'node_modules/')
     if (fs.existsSync(modulesPath)) {
-      res = findModulesInDir(modulesPath + '/', keyword).sort(prioritize)
+      findModulesInDir(modulesPath, keyword).forEach(m => modules.add(m))
     }
   }
-
-  res.push.apply(
-    res,
-    findModulesInDir(__dirname + '/../node_modules/', keyword)
-      .filter(module => {
-        return !res.find(m => {
-          return m.module == module.module
-        })
-      })
-      .sort(prioritize)
-  )
-
-  return res
+  // look in configPath for modules
+  if (app.config.configPath !== app.config.appPath) {
+    searchAddModules(app.config.configPath)
```

Comment:
For some reason `searchAddModules` is not very intuitive for me. Would it make sense that this line would read `findModules(app.config.configPath, keyword).forEach(m => modules.add(m))` ? Then `findModules(dir, keyword)` could be static, no side effects.

Mergeable as it is, this is no biggie.